### PR TITLE
Update t1_coronavirus_symptom_tracker.md

### PR DESCRIPTION
**Pull request by The-Kergan**
1. Approve change after T10 live on site
2. Link needed to T10 in text added

### CODE REVIEW

File: docs/f2-living-in-a-household-with-someone-with-coronavirus-symptoms.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
 # F2: Living in a Household with Someone with Coronavirus Symptoms
 
-*Coming 3 April 2020*
\ No newline at end of file
+*Coming 3 April 2020*
```
Comment (The-Kergan) [inline docs/f2-living-in-a-household-with-someone-with-coronavirus-symptoms.md:3]:
```

```

File: docs/research-process.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Research Process Infographic
+https://user-images.githubusercontent.com/44531202/78272502-97b36e00-74db-11ea-95bf-9acaff08b3b4.jpg
```
Comment (The-Kergan) [inline docs/research-process.md:2]: @geoffwasham this image needs to be saved in public vuepress folder

File: docs/research-process.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+**LOCATION: Linked from a button on 'About' page > just above 'References' section.**
```
Comment (The-Kergan) [inline docs/research-process.md:3]: @geoffwasham this text is not needed on the page - it is an instruction on where to upload the image

File: docs/research-process.md (hunk 2 above)
Comment (geoffwasham) [reply docs/research-process.md:2]: I fixed this in another branch before seeing this convo.

File: docs/research-process.md (hunk 3 above)
Comment (geoffwasham) [reply docs/research-process.md]: Fixed.

**Pull request merged**